app/src/main/res/values/strings.xml

```diff
@@ -5,8 +5,8 @@
     <string name="action_search">Search</string>
     <string name="action_add">Add</string>
 
-    <string name="noGiftCards">You don\'t have any cards at the moment. Click the "+" (plus) button to get started.\n\nCatima lets you carry your cards on your phone, so they are always within reach.</string>
-    <string name="noMatchingGiftCards">No cards match the filter. Please try some different terms.</string>
+    <string name="noGiftCards">Click the "+" (plus) button to add a card first.\n\nCatima carries your cards on the device, so they are always within reach.</string>
+    <string name="noMatchingGiftCards">Could not find any such cards. Please try changing your search.</string>
```

Comment:
```suggestion
    <string name="noMatchingGiftCards">Maybe if you look elsewhere, or for something else you will find what you are looking for.</string>
```

Reply:
Well now it's just something you would find in a fortune cookie :P

Reply:
I don't disagree, but I can't think of anything that covers both.

Reply:
Could split it into 2 different strings and display "This group is empty" when the group is empty. Then we can keep this one search-related.

Reply:
It still is somehow. The user knows the user carried out a search and got this. Can be change later I think.